CMakeLists.txt

```diff
@@ -316,11 +337,9 @@ endif()
 #-----------------------------------------------------------------------------
 # Testing
 #-----------------------------------------------------------------------------
-if(CMAKE_PROJECT_NAME STREQUAL HERMES)
+if(NOT HERMES_EXTERNALLY_CONFIGURED AND BUILD_TESTING)
```

Comment:
Can you explain what the `HERMES_EXTERNALLY_CONFIGURED` option is for? I've never found it useful for anything.

Reply:
Coverage is done only when we are building hermes, but not when using it as a dependency of another library.